docker/Makefile

```
@@ -82,6 +82,15 @@ lint:
 upd-dev:
 	$(dc) up -d app
 
+stop-prod-celery:
+	$(dcp) stop celery-beat celery-command-worker celery-default-worker celery-mail-worker
+
+upd-prod-celery:
+	$(dcp) up -d celery-beat celery-command-worker celery-default-worker celery-mail-worker
+
+upd-prod:
```

instead of `... && $(MAKE) upd-prod-celery`
```suggestion
upd-prod: upd-prod-celery
```